lib/facter/monit_version.rb

```diff
@@ -0,0 +1,14 @@
+# A facter fact to determine the installed version of monit.
+
+module Facter::Util::MonitVersion
+  class << self
+    def get_monit_version
+      monit_version = Facter::Util::Resolution.exec('monit -V 2>&1')
```

Comment:
Facter would complain via STDERR that it couldn't find the command, that's why the which is there.

Perhaps it is a version thing with Facter.. just noticing that this code does not cause `command not found` issues.